cypress/integration/admin.js

```diff
@@ -6,14 +6,34 @@ const check = new Checks();
 const participantNum = 43; //later change to change from hard coded version
 const clientNum = 84;
 const taskNum = 499;
+const projNum = 132;
+
+var randParticNum = Math.round(Math.random() * (participantNum - 1));
+var randTask = Math.round(Math.random() * taskNum); //TO DO to make not hardcoded 
+var randClient = Math.round(Math.random() * clientNum); //TO DO get the length from DOM
+var randProject =Math.round(Math.random() * projNum);
+
+function isTimeLogged(nameTitle) {
+    return cy.get(nameTitle).should('exist')
+}
+
+function loggedTime(nameTitle) {
+    if(isTimeLogged(nameTitle)) {
+        return cy.get(nameTitle).text();
+    }
+    else {
+        return 0;
+    }
+}
+
 
 describe('Sourcebooks admin', function() {
     beforeEach(() => {
         cy.loginAs("Admin");
         login.visit();
     })
 
-    it('Admin should be able to create a task', function() {
+    /*it('Admin should be able to create a task', function() {
```

Comment:
pull request should not contain any commented out test cases, it should be ether deleted or un-commented